### PR TITLE
fix: Study programme choice

### PR DESCRIPTION
**Pull request by mrshu**
* Previously it was enough to put a study programme abbreviation into
  AIS for it to get filled in automatically. This is no longer the case
  now, as for instance `FYZ` matches both `FYZ` and `FYZ/k`.

* This commit fixes the situation by automatically choosing the
  pre-filled (first) option when such a dialog gets opened while filling
  in the study programme.

* The PSC field in the dialog that chooses it in the address fileds has
  changed position. This commit introduces a change that reflects this
  situation.

Signed-off-by: mr.Shu <mr@shu.io>

### CODE REVIEW

File: eprihlaska/ais_utils.py [hunk 1]
```diff
@@ -399,6 +414,8 @@ def save_application_form(ctx,
     with app.collect_operations() as ops:
         app.d.enterButton.click()
 
+    print("Post submit: {}".format(ops), file=sys.stderr)
```
Comment (Adman) [inline eprihlaska/ais_utils.py:417]: Do we need these `print` statements, or they are here just for easier debugging failed requests?

**Review by Adman — APPROVED** [review]

File: eprihlaska/ais_utils.py (hunk 1 above)
Comment (mrshu) [reply eprihlaska/ais_utils.py:417]: These are indeed just for debugging

Comment (tvinar) [reply eprihlaska/ais_utils.py:417]: Is it such a big problem to leave them in? (After all, more diagnostic info leads to easier debugging...)

Comment (mrshu) [reply eprihlaska/ais_utils.py:417]: @tvinar what Adman was probably alluding to is the fact that this is strictly "the wrong" way of doing it. That said, I do think we can live with this for sure.

**Pull request merged**